enterprise/frontend/src/embedding-sdk/components/public/dashboard/EditableDashboard/EditableDashboard.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  DASHBOARD_EDITING_ACTIONS,
+  SDK_DASHBOARD_VIEW_ACTIONS,
+} from "metabase/dashboard/components/DashboardHeader/DashboardHeaderButtonRow/constants";
+import { DASHBOARD_ACTION } from "metabase/dashboard/components/DashboardHeader/DashboardHeaderButtonRow/dashboard-action-keys";
+import type { MetabasePluginsConfig as InternalMetabasePluginsConfig } from "metabase/embedding-sdk/types/plugins";
+import { getEmbeddingMode } from "metabase/visualizations/click-actions/lib/modes";
+import { EmbeddingSdkMode } from "metabase/visualizations/click-actions/modes/EmbeddingSdkMode";
+
+import {
+  SdkDashboard,
+  type SdkDashboardInnerProps,
+  type SdkDashboardProps,
+} from "..";
+
+export type EditableDashboardProps = SdkDashboardProps;
+
+export const EditableDashboard = (props: SdkDashboardProps) => {
+  const dashboardActions: SdkDashboardInnerProps["dashboardActions"] = ({
+    isEditing,
+    downloadsEnabled,
+  }) =>
+    isEditing
+      ? DASHBOARD_EDITING_ACTIONS
+      : downloadsEnabled.pdf
+        ? [...SDK_DASHBOARD_VIEW_ACTIONS, DASHBOARD_ACTION.DOWNLOAD_PDF]
+        : SDK_DASHBOARD_VIEW_ACTIONS;
+
+  const getClickActionMode: SdkDashboardInnerProps["getClickActionMode"] = ({
+    question,
+  }) =>
+    getEmbeddingMode({
+      question,
+      queryMode: EmbeddingSdkMode,
+      plugins: props.drillThroughQuestionProps
+        ?.plugins as InternalMetabasePluginsConfig,
+    });
+
+  return (
+    <SdkDashboard
+      {...props}
+      dashboardActions={dashboardActions}
+      getClickActionMode={getClickActionMode}
```

Comment:
nit: the other 2 dashboard components, put `getClickActionMode` first

```suggestion
      getClickActionMode={getClickActionMode}
      dashboardActions={dashboardActions}
```

Reply:
done 🙂